yarn.config.js

```diff
@@ -9,21 +9,26 @@ const {defineConfig} = require(`@yarnpkg/types`);
  * @typedef {import('@yarnpkg/types').Yarn.Constraints.Dependency} Dependency
  */
 
+const IGNORE_CONSISTENT_DEPENDENCIES_FOR = new Set([
+  `.`,
```

Comment:
Temporary until I fix the docusaurus deps to be consistent with everything else.